src/hooks/useTopRatedTVshows.ts

```diff
@@ -1,10 +1,14 @@
-import { useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
 import { useEffect } from "react";
 import { API_OPTIONS } from "../utils/constants";
 import { addTopRatedTVshows } from "../utils/moviesSlice";
+import { RootState } from "../utils/appStore";
 
 export const useTopRatedTVshows = () => {
   const dispatch = useDispatch();
+  const topRatedTVshows = useSelector(
+    (store: RootState) => store.movies.topRatedTVshows
+  );
 
```

Comment:
Add a try catch for all the endpoint calls, if something fails it shouldn't crash the app. Instead should be handle and show the message. Same goes for all the hooks.

Reply:
done